tests/test_transforms/test_inference/test_transform.py

```diff
@@ -606,6 +608,7 @@ def _test_transform_train_new_segments(self, ts, transform, train_segments, expe
                 {"create": {"res_1", "res_2", "res_3", "res_4"}},
             ),
             (HolidayTransform(out_column="res"), "regular_ts", {"create": {"res"}}),
+            (HolidayTransform(out_column="res", mode="category"), "regular_ts", {"create": {"res"}}),
```

Comment:
Seems a little bit inconsistent that in some places you set `mode="binary"`. May be let's set it in all places in inference tests.